docs/articles_en/openvino_workflow/torch_compile.rst

```diff
@@ -68,20 +68,56 @@ To use ``torch.compile``, you need to add an import statement and define one of
          :align: center
 
 
-Environment Variables
-+++++++++++++++++++++++++++
 
-* **OPENVINO_TORCH_BACKEND_DEVICE**: enables selecting a specific hardware device to run the application. 
-  By default, the OpenVINO backend for ``torch.compile`` runs PyTorch applications using the CPU. Setting 
-  this variable to GPU.0, for example, will make the application use the integrated graphics processor instead.
-* **OPENVINO_TORCH_MODEL_CACHING**: enables saving the optimized model files to a hard drive, after the first application run.
-  This makes them available for the following application executions, reducing the first-inference latency.
-  By default, this variable is set to ``False``. Setting it to ``True`` enables caching.
-* **OPENVINO_TORCH_CACHE_DIR**: enables defining a custom directory for the model files (if model caching set to ``True``).
-  By default, the OpenVINO IR is saved in the ``cache`` sub-directory, created in the application's root directory. 
+Options
+++++++++++++++++++++
+
+You can use a dictionary with options for ``torch.compile`` to set the backend device, 
```

Comment:
Can we make sure to reflect that these options are optional ? 
Something like, 
You can use an optional dictionary of options for torch.compile to set the backend_device, .... Don't know if it's confusing with word options. :)

Reply:
```suggestion
You can use an optional dictionary of options for ``torch.compile`` to set the backend device, 
```

Reply:
We'll rephrase it to stress that options are not mandatory and make the "dictionary" less pronounced.